### ISSUE TITLE
Magithub on melpa stable should use a stable version of s

### ISSUE THREAD

**Issue by SerialVelocity** [vermiculus/magithub#169]
I can't currently install magithub because the version of s used is not present in stable melpa.

```
magithub is an incompatible package.

     Status: Incompatible because it depends on uninstallable packages.
    Archive: melpa-stable

    Version: 0.1.4
    Summary: Magit interfaces for GitHub
   Requires: emacs-25, magit-2.8.0, s-20170428.1026 (not available), ghub+-0.2
   Homepage: https://github.com/vermiculus/magithub
   Keywords: git tools vc 
```

Could you please depend on a stable version and release a new minor version?

**Comment by vermiculus** [comment]
It's already in the develop branch. I'll release when that dev is stable.

**Comment by SerialVelocity** [comment]
Thanks!

**Comment by jiacai2050** [comment]
Same issue here. How can I solve this by now ?

**Comment by vermiculus** [comment]
I don't know of a way as I don't use melpa stable. You may be able to pin magithub to use the regular melpa?

Otherwise I just need to make the time to merge into master.

**Comment by SerialVelocity** [comment]
I manually edited the melpa repo file and changed the version of 's'

**Comment by jiacai2050** [comment]
Thank your guys.

**Comment by vermiculus** [comment]
This should be fixed now with the push of 0.1.5. This will be the last release before 0.2.

**Comment by jiacai2050** [comment]
@vermiculus  I upgrade to 0.1.5, then shortcuts `H` disappear in magit-status buffer, I came across this problem a few times, How can I debug this ?

**Comment by vermiculus** [comment]
@jiacai2050 Can you be more specific? Maybe provide an example of what you see vs. what you expect to see? [`magithub-dispatch-popup` definitely still exists.](https://github.com/vermiculus/magithub/blob/c13d6cb79d2ab2b2fac2f0292c2907ff35fc7fca/magithub.el#L61)

**Comment by jiacai2050** [comment]
I find this error in Message buffer
```
Error: (error "Keyword argument nil not one of (:query :payload :headers :unpaginate :noerror :reader :username :auth :host)")
Keyword argument nil not one of (:query :payload :headers :unpaginate :noerror :reader :username :auth :host)
Magithub features not configured: (pull-request-merge pull-request-checkout); see variable ‘magithub-features’ to turn off this message

```
Is this relate to my issue ?

**Comment by vermiculus** [comment]
What command are you trying to run? This is a ghub versioning issue.
